.github/workflows/test-suite.yml

```diff
@@ -71,6 +71,37 @@ jobs:
             tests/coverage.xml:clover
             coverage/clover.xml:clover
 
+  Cypress:
+    runs-on: ubuntu-latest
+    services:
```

Comment:
i'm not sure if this is needed for `cypress` tbh @m-triassi

Reply:
if you mean the `services` block, then thats definitely required, the tests need to be able to read and write data to a database. if you mean the `runs-on` line, that's also required, we need to define what OS to run the pipeline on

Reply:
yeah i was just wondering if the service block was needed

Reply:
i see, yeah services is definitely needed